media-libs/fontconfig/patches/fontconfig-2.13.1.patchset

```diff
@@ -0,0 +1,80 @@
+From 56443bc7c0c6164fe6bb5b005b40b32da85743de Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Tue, 28 May 2019 16:50:19 +0200
+Subject: Add Haiku to configure.ac, disable test 106632
+
+
+diff --git a/configure.ac b/configure.ac
+index fb8af46..d780985 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -92,10 +92,14 @@ case "$host" in
+   *-*-mingw*)
+     os_win32=yes
+     ;;
++  *-*-haiku*)
++    os_haiku=yes
++    ;;
+   *)
+     os_win32=no
+ esac
+ AM_CONDITIONAL(OS_WIN32, test "$os_win32" = "yes")
++AM_CONDITIONAL(OS_HAIKU, test "$os_haiku" = "yes")
+ 
+ dnl ==========================================================================
+ dnl gettext stuff
+diff --git a/test/Makefile.am b/test/Makefile.am
+index 79bcede..98908d7 100644
+--- a/test/Makefile.am
++++ b/test/Makefile.am
+@@ -79,6 +79,7 @@ test_hash_CFLAGS = -I$(top_builddir) -I$(top_builddir)/src $(UUID_CFLAGS)
+ test_hash_LDADD = $(UUID_LIBS)
+ TESTS += test-hash
+ 
++if !OS_HAIKU
+ check_PROGRAMS += test-bz106632
+ test_bz106632_CFLAGS =					\
+ 	-I$(top_builddir)				\
+@@ -90,6 +91,7 @@ test_bz106632_CFLAGS =					\
+ 	$(NULL)
+ test_bz106632_LDADD = $(top_builddir)/src/libfontconfig.la
+ TESTS += test-bz106632
++endif
+ 
+ EXTRA_DIST=run-test.sh run-test-conf.sh $(TESTDATA) out.expected-long-family-names out.expected-no-long-family-names
+ 
+-- 
+2.21.0
+
+
+From d56c55e2d1e0a75201ba5214d2c6b3c12e8ed61f Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Thu, 30 May 2019 09:15:09 +0200
+Subject: gcc2 fix
+
+
+diff --git a/src/fccfg.c b/src/fccfg.c
+index d7c48e8..8321d6e 100644
+--- a/src/fccfg.c
++++ b/src/fccfg.c
+@@ -2228,6 +2228,8 @@ FcConfigRealFilename (FcConfig		*config,
+     const FcChar8 *sysroot = FcConfigGetSysRoot (config);
+     FcChar8 *n = FcConfigFilename (url);
+     FcChar8 *nn = NULL;
++    FcChar8 *dirname = FcStrDirname (nn);
++    FcChar8 *path;
```

Comment:
this should be moved in next-next block. dirname is already defined somewhere?

Reply:
Moved it to the next block, now the test for 106632 also works fine, so the patch for Haiku isn't needed anymore